getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
+
+- **Timestamp**: When the event occurred.
+- **Attributes**: A set of key-value pairs offering details about the event context.
+- **Metadata**: Contextual labels and IDs, such as trace ID and span ID, that connect related events.
+
+Rather than simply calling them logs or metrics, event data reflects a broader range of interactions, crossing boundaries from engineering to product management, security, and beyond. This includes everything from the number of documents created in a product to the sequence of API calls during a checkout process.
```

Comment:
Good. 

"Whole engineers might most often work with timestamped event data in the form of logs or metrics..."

Something like this ^ might be more effective. I don’t love the phrasing, but the goal again would be to say: We know our most common reader is likely to be an engineer because of Axiom’s history, but that there is a bigger opp here for a business.